swap_meet/clothing.py

```diff
@@ -1,2 +1,15 @@
-class Clothing:
-    pass
\ No newline at end of file
+from uuid import uuid4
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, id = None, fabric = None, condition = 0):
+        super().__init__(id = id, condition = condition)
```

Comment:
When you pass named arguments you don't put spaces around `=`.  This same comment applies to all other places in your project where you have spaces surrounding = for named args.

```suggestion
        super().__init__(id=id, condition=condition)
```

However, given that the constructor only takes 2 simple arguments, I feel like you can leave of the named arguments completely. 
```suggestion
        super().__init__(id, condition)
```

Check out this [Stack Overflow thread](https://stackoverflow.com/a/7041991) about when or when not to use named arguments.